lib/modules/ens/embarkjs.js

```diff
@@ -226,6 +226,29 @@ __embarkENS.lookup = function (address, callback) {
   });
 };
 
+__embarkENS.registerSubDomain = function (name, address, callback) {
```

Comment:
Why is there 2 functions with the same definition?

Reply:
Good question. That's probably a rebase oopsie. I was also wondering when I saw that I "added" the function in this PR. 
I'll remove.